### PR TITLE
N2WS Tile - Typos and Indentation fixes

### PR DESCRIPTION
**Pull request by eliadeini**
### What does this PR do?

Fixing typos and indentation in N2WS tile, by fixing manifest.json and README.md

### Motivation

Make our tile look better :-)

### Review checklist

- [X] PR has a fix for N2WS tile (https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: n2ws/manifest.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
```
Comment (hithwen) [inline n2ws/manifest.json:29]: The keys of this json cannot be changed once the integration tile has been deployed (as is the case). However this is not visible in app. If you want to change the title you can do it on the dashboard json itself.

Comment (hithwen) [inline n2ws/manifest.json:29]: Once deployed to prod these names cannot be changed, however they are not visible inside the app and you can change the title inside the dashboard json.

**Comment by apigirl** [comment]
hi there! we really appreciate you pointing out the issue with the tile layout here. your changes actually won't fix it for us though. I am going to create an internal issue for us to take a look here and push a fix, so thanks for letting us know! For now, your updates to the readme should not be merged.

**Comment by eliadeini** [comment]
> hi there! we really appreciate you pointing out the issue with the tile layout here. your changes actually won't fix it for us though. I am going to create an internal issue for us to take a look here and push a fix, so thanks for letting us know! For now, your updates to the readme should not be merged.

Hi, thanks for your comment. When I submitted the README file I didn't really understand the layout, so things became a bit messy, like the code snippets, the numeration and the paragraph, and a small typo on the manifest file ('N2ws' instead of 'N2WS') . After I made this merge request and got a comment I noticed that redundant commits came in, it was not my intention, planned to fix it today.
What should I do now in order to fix this issue? Just close the PR or leave it as is?
Thanks

**Comment by hithwen** [comment]
Hi @eliadeini the commit history on PRs does not matter much since we squash them on merge. You can leave it as it is

**Comment by eliadeini** [comment]
> Hi @eliadeini the commit history on PRs does not matter much since we squash them on merge. You can leave it as it is or close an open a new one, whatever works best for you.

Hi @hithwen, we are still seeing the typos I tried to fix on N2WS integration tile. What can I do in order to fix it? 
Our customers are being exposed to that and it something that it should be easy to fix.

**Comment by irabinovitch** [comment]
@eliadeini Would you be able to resolve the conflicts on this PR?

**Comment by alexvetras** [comment]
Hi @eliadeini, I see there are three dashboards .json files, but the dashboards have the same titles. Are these dashboards meant to represent different views?

**Comment by alexvetras** [comment]
@eliadeini, will you be adding out-of-the-box monitors? (https://docs.datadoghq.com/monitors/monitor_types/). We request these for each integration to help users alert on issues.

File: n2ws/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ and more by Datadog monitoring service. This integration allows users to monitor
 
 ### Installation
 
-1. Install the [Python integration][1].
+1.	##### Install the [Python Integration][1]
 
 2. Enable support for Datadog on your N2WS instance:
```
Comment (coignetp) [inline n2ws/README.md:16]: This section is duplicated

**Review by coignetp — CHANGES_REQUESTED** [review]
Just a nit

**Comment by alexvetras** [comment]
@eliadeini, small suggestion that is not a blocker, but it would be good to add an image or two to the README to display in the overview. Maybe of a dashboard?

**Comment by eliadeini** [comment]
> @eliadeini, small suggestion that is not a blocker, but it would be good to add an image or two to the README to display in the overview. Maybe of a dashboard?

Yes, it is a good idea. How can I do it?

**Comment by alexvetras** [comment]
Hi @eliadeini, you can just add an image to your README file and it will populate in the tile's overview section. Thanks!

**Comment by eliadeini** [comment]
> Hi @eliadeini, I see there are three dashboards .json files, but the dashboards have the same titles. Are these dashboards meant to represent different views?

One of them emphasizes the differences between the successful backups of each kind of AWS entity/service backed up by N2WS, the other two are similar and more general, except for the kind of diagrams they are showing.
I'll change the titles accordingly.

File: n2ws/README.md (hunk 2 above)
Comment (eliadeini) [reply n2ws/README.md:16]: OOPS, my mistake.
Fixing

**Comment by eliadeini** [comment]
> Hi @eliadeini, you can just add an image to your README file and it will populate in the tile's overview section. Thanks!

Hi Alex, 
I fixed the comments you gave about the text (phrase duplication in instructions and new titles for the dashboards).
Would you mind except these textual changes? I'll add images and monitor next week in another PR, after I'll finish an urgent task I should do.

**Comment by FlorianVeaux** [comment]
> I fixed the comments you gave about the text (phrase duplication in instructions and new titles for the dashboards).
Would you mind except these textual changes? I'll add images and monitor next week in another PR, after I'll finish an urgent task I should do.

Hey @eliadeini, sounds good I'll merge and update the tile as is for now and you can open another PR in the future for adding images and monitor

You should see the changes applied in the next 30min or so

**Review by FlorianVeaux — APPROVED** [review]

**Comment by FlorianVeaux** [comment]
I missed that you re-renamed dashboards in your last commit. As mentioned by @hithwen you can't do that: https://github.com/DataDog/integrations-extras/pull/760#discussion_r525258661
But the ids are not visible in-app so that shouldn't be a problem though, right?

I'll need to get this fixed before being able to sync that to production. I'll make another PR

**Comment by apigirl** [comment]
@FlorianVeaux yeah, this adds a bunch of really odd formatting to the docs page that looks strange in the docs site. We had actually already fixed the broken tile formatting. I'll create a PR to undo the stuff that doesn't work in our site.

**Comment by apigirl** [comment]
tile was fixed in https://github.com/DataDog/integrations-extras/pull/764 - reverting to that in https://github.com/DataDog/integrations-extras/pull/770

**Comment by apigirl** [comment]
we just investigated, and the person who normally does the tile sync was out on vacation and then out sick, so #764 didn't get synced after it got merged in. Everything is updated and synced now based on the reversion to the fix in #764. Thanks again for letting us know there was an issue!

**Comment by eliadeini** [comment]
> we just investigated, and the person who normally does the tile sync was out on vacation and then out sick, so #764 didn't get synced after it got merged in. Everything is updated and synced now in the tile and the documentation based on reverting back to the fix in #764. Thanks again for letting us know there was an issue!

Hi @apigirl , I found another strange issue: In the title "View summary data from all the connected N2WS Backup & Recovery hosts", the N2WS is written as N2wS. I've already fixed it but my fix didn't influence the tile.
Can you please take a look?

**Comment by FlorianVeaux** [comment]
Ok I think we should be all set now: 
image

Let us know if there's anything left to change

**Comment by eliadeini** [comment]
> Ok I think we should be all set now:
> image
> 
> Let us know if there's anything left to change

Thanks, it is OK for the meantime. I'll probably add a monitor and images to our README file as you suggested in another PR.
How can I add images to the README file? should I have a URL that directs to the image, or it can just be a file on my computer and it will be uploaded to DataDog somehow?

**Comment by FlorianVeaux** [comment]
You can have it work this way: https://github.com/DataDog/integrations-extras/tree/master/appkeeper
Put your images into `integrations-extras/n2ws/images/*` and reference them as images in the ReadMe using such a link:
`https://raw.githubusercontent.com/DataDog/integrations-extras/master/n2ws/images/integration.jpg`

**Comment by eliadeini** [comment]
> You can have it work this way: https://github.com/DataDog/integrations-extras/tree/master/appkeeper
> Put your images into `integrations-extras/n2ws/images/*` and reference them as images in the ReadMe using such a link:
> `https://raw.githubusercontent.com/DataDog/integrations-extras/master/n2ws/images/integration.jpg`

Good idea, I'll do it.
Thanks

**Comment by apigirl** [comment]
@eliadeini - cool, thank you for adding the image!

**Pull request merged**